File: algorithm/src/main/kotlin/ja/ch01/Max3.java
package ja.ch01;

public class Max3 {
    public int max3(int a, int b, int c) {
        int max = a;
        if (max < b) {
            max = b;
        }
        if (max < c) {
            max = c;
        }

        return max;
    }
}

File: algorithm/src/main/kotlin/ko/ch01/Max3.kt
package ko.ch01

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val a = br.readLine().toInt()
    val b = br.readLine().toInt()
    val c = br.readLine().toInt()
    var max = a
    if (max < b) max = b
    if (max < c) max = c

    println("Max is $max")
}